gamemodes/terrortown/entities/entities/ttt_base_placeable.lua

```diff
@@ -392,4 +392,43 @@ if SERVER then
 
         return true
     end
-end
+end -- SERVER
+
+if CLIENT then
+    ---
+    -- This is triggered, when you focus a marker of an entity and press 'Use'-Key
+    -- Override to handle remote use
+    -- @realm client
+    function ENT:RemoteUse() end
```

Comment:
I dont get the Override part. This is more a OnRemoteUse for sure, but whats with that UseOverride?

Reply:
Because it is a hook that is intended to be overriden. In general I prefer the `OnXYZ` naming as well, but the other use hook already has the name `UseOverride`, hence I shuggested that as a name

Reply:
Comin back to this, as Im gonna remove UseOverride in my cleanup, I dont think it should be called OnXYZ.
The gmod hook is also just called Use and is called on using it.
You can always call it yourself, so it's not even wrong to not be called OnXYZ or Override, because hooks are always called and can be overriden.

Reply:
@TimGoll this conversation is still open but i think it was solved. Could you double check?

Reply:
yep